.ci_support/win_64_CLANG_VERSION17.0.6CL_VERSION19.29.30139RUNTIME_VERSION14.29.30139VCVER14.2VSYEAR2019.yaml

```diff
@@ -1,23 +1,15 @@
 CLANG_VERSION:
 - 17.0.6
 CL_VERSION:
-<<<<<<<< HEAD:.ci_support/win_64_CLANG_VERSION17.0.6CL_VERSION19.29.30139RUNTIME_VERSION14.29.30139VCVER14.2VSYEAR2019.yaml
 - 19.29.30139
-========
-- 19.41.34120
->>>>>>>> upstream/main:.ci_support/win_64_CLANG_VERSION19.1.0.rc2CL_VERSION19.41.34120RUNTIME_VERSION14.41.34120VCVER14.3VSYEAR2022channel_targetsconda-forge_llvm_rc.yaml
 FINAL_CFLAGS:
```

Comment:
Please try not to commit conflicts where possible. It's pretty easy to avoid with either:
```
git checkout --theirs .ci_support/  # or --ours
```
or 
```
rm -rf .ci_support/  # in case of file renames present in the conflict
git checkout upstream/main -- .ci_support/
git add .
```
before committing the merge.